scroll-animations-1/Overview.bs

```diff
@@ -736,67 +736,79 @@ spec: cssom-view-1; type: dfn;
 	together in a single declaration.
-	If the <<'animation-delay-end'>> value is omitted,
+	If the <<'animation-range-end'>> value is omitted,
 	it is set to zero.
 	If a <<timeline-range-name>> alone is specified,
-	'animation-delay-start' is set to that name plus ''0%''
-	and 'animation-delay-end' is set to that name plus ''100%''.
+	'animation-range-start' is set to that name plus ''0%''
+	and 'animation-range-end' is set to that name plus ''100%''.
 
 	<pre class="propdef">
-		Name: animation-delay-start
-		Value: [ <<time>> | <<timeline-range-name>> <<percentage>> ]#
-		Initial: 0s
+		Name: animation-range-start
+		Value: [ normal | <<timeline-range-name>> <<percentage>> ]#
+		Initial: normal
 		Applies to: all elements
 		Inherited: no
 		Percentages: relative to the specified [=named timeline range=]
-		Computed value: list, each item either a duration or a timeline range and progress percentage
+		Computed value: list, each item either the keyword ''normal'' or a timeline range and progress percentage
 		Animatable: no
 	</pre>
 
-	Defines a delay between the start of the animation
-	and when it begins executing
-	(i.e. when keyframes attached to 0% progress are mapped).
+	Shifts the start of the animation
+	(i.e. when keyframes attached to 0% progress are mapped)
+	to the specified position on the timeline.
 
-	<dl dfn-for="animation-delay-start, animation-delay-end" dfn-type=value>
-		<dt><dfn><<time>></dfn>
+	<dl dfn-for="animation-range-start" dfn-type=value>
+		<dt><dfn>auto</dfn>
 		<dd>
-			Defines the delay as a duration. See [[CSS-ANIMATIONS-1]].
+			The start of the animation attaches to the start of the timeline.
+
 		<dt><dfn><<timeline-range-name>> <<percentage>></dfn>
 		<dd>
-			Calculates the delay as the difference between
-			the start of the animation
-			and the specified point on the [=timeline=].
+			The animation starts at the specified point on the [=timeline=],
+			plus any applicable start delay.
 	</dl>
 
 	<pre class="propdef">
-		Name: animation-delay-end
-		Value: [ <<time>> | <<timeline-range-name>> <<percentage>> ]#
-		Initial: 0s
+		Name: animation-range-end
+		Value: [ normal | <<timeline-range-name>> <<percentage>> ]#
+		Initial: normal
 		Applies to: all elements
 		Inherited: no
 		Percentages: relative to the specified [=named timeline range=]
-		Computed value: list, each item either a duration or a timeline range and progress percentage
+		Computed value: list, each item either the keyword ''normal'' or a timeline range and progress percentage
 		Animatable: no
 	</pre>
 
-	Defines a delay between the end of the animation
-	and when it ends executing
-	(i.e. when keyframes attached to 100% progress are mapped).
-	Values are as for 'animation-delay-start',
-	but calculating backwards from the end,
-	i.e. a ''2s'' delay effectively shortens the animation duration,
-	whereas ''-2s'' lengthens it.
+	Shifts the end of the animation
+	(i.e. when keyframes attached to 100% progress are mapped)
```

Comment:
This is true if iteration count = 1 and end delay is 0, but if iteration count > 1 there will be multiple points at which 100% is reached and if end delay is > 0 100% will be reached before animation-range-end. This is the [end time](https://drafts.csswg.org/web-animations-1/#end-time) of the animation.

Reply:
OK, I *think* I fixed this.